components/notification-generator/minecraftTextParser.tsx

```
@@ -0,0 +1,117 @@
+  if (htmlOutput) {
+    return (
+      <span className="font-minecraft" dangerouslySetInnerHTML={{ __html: htmlOutput }} />
+    );
+  }
+
+  return <>{renderLegacy(cleanedText)}</>;
+};
+
+function sanitizeInput(input: string): string {
+  if (!input) return '';
+  return input.replace(/\x00|\u0000|[\r\v\u2028\u2029]/g, '');
```

Biome (1.9.4)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)